ci/install_protobuf.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+[ -z "${PROTOBUF_VERSION}" ] && export PROTOBUF_VERSION="3.11.4"
+
+cd /
+wget https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-cpp-${PROTOBUF_VERSION}.tar.gz
```

Comment:
Not using apt?

Reply:
I've found it challenging before to install both the protobuf-dev libraries and protoc compiler using apt.

Reply:
Let's keep it as-is in this PR.